db/migrate/20190522165029_fix_reports_table_pkey_sequence.rb

```diff
@@ -0,0 +1,8 @@
+class FixReportsTablePkeySequence < ActiveRecord::Migration
+  def up
+    execute "SELECT SETVAL('ministries_id_seq', (SELECT MAX(id) FROM ministries));" 
+  end
+
+  def down
```

Comment:
Just as a good practice we can `raise ActiveRecord::IrreversibleMigration` inside `#down`, because there is no way to reverse and get back to the same db state as before.

Reply:
I thought about that, but its really a harmless migration.

Will add the raise then ⬆️

Reply:
Yeah I think you're right that it doesn't really matter in this particular case. I guess it's just a habit I have for writing migrations!